ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/StatementRewriter.java

```diff
@@ -222,7 +223,12 @@ protected AstNode visitKsqlWindowExpression(
 
     @Override
     protected AstNode visitTableElement(final TableElement node, final C context) {
-      return node;
+      return new TableElement(
+          node.getLocation(),
```

Comment:
How do the location and namespace relate to the alias type? I mean, why is this change introduced in this PR?